[DOC] Add a node to local install (#5040)
pachyderm/pachyderm: doc/docs/1.10.x/getting_started/local_installation.md

some of the Pachyderm basics and is not designed to be a production
environment.

> When you run `pachctl deploy local`, a single-node cluster is created.

Reviewer: ```suggestion
`pachctl deploy local` is designed for a single-node cluster.
```
